charts/alertmanager/templates/alertmanager-statefulset.yaml

```diff
@@ -107,20 +103,8 @@ spec:
           - containerPort: {{ .Values.global.authSidecar.port }}
             name: auth-proxy
             protocol: TCP
-          readinessProbe:
-            httpGet:
-              path: /healthz
-              port: {{ .Values.global.authSidecar.port }}
-              scheme: HTTP
-            initialDelaySeconds: 10
-            periodSeconds: 10
-          livenessProbe:
-            httpGet:
-              path: /healthz
-              port: {{ .Values.global.authSidecar.port }}
-              scheme: HTTP
-            initialDelaySeconds: 10
-            periodSeconds: 10
+          readinessProbe: {{- tpl (toYaml .Values.readinessProbe) . | nindent 12 }}
+          livenessProbe: {{ tpl (toYaml .Values.livenessProbe) . | nindent 12 }}
```

Comment:
This `tpl` technique lets us embed helm logic in values.yaml so we can access global values.